Examples/Algorithms/AmbiguityResolution/src/ScoreBasedAmbiguityResolutionAlgorithm.cpp

```diff
@@ -116,8 +111,7 @@ ActsExamples::ScoreBasedAmbiguityResolutionAlgorithm::execute(
   const auto& tracks = m_inputTracks(ctx);  // Read input data
   ACTS_VERBOSE("Number of input tracks: " << tracks.size());
 
-  Acts::ScoreBasedAmbiguityResolution::OptionalCuts<ConstTrackProxy>
-      optionalCuts;
+  Acts::ScoreBasedAmbiguityResolution::Optionals<ConstTrackProxy> optionalCuts;
```

Comment:
```suggestion
  Acts::ScoreBasedAmbiguityResolution::Optionals<ConstTrackProxy> optionals;
```

Or was it intentional to keep it here with `Cuts`?

Reply:
No, I forgot to change the name . Thanks for noticing.